### PR TITLE
check out my code

### PR DESCRIPTION
**Pull request by GuderianE**

### CODE REVIEW

File: Elias/exercise-4.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// Are apples and oranges equal?
+console.log(apples === oranges); //==> false
+
+//Are apples and oranges unequal?
+console.log(apples !== oranges); //==> true
+
+// are apples greater than oragnes?
+console.log(apples > oranges); //==> false
+
+// are apples less than oranges?
```
Comment (ciamac-da) [inline Elias/exercise-4.js]: Here also try to check if they are EQUAL(less or equal) 
means <= instead of <

File: Elias/exercise-4.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// is mangos added to oranges greater than mangoes multiplied by apples?
+let sumApMa = apples * mangos;
+let sumMaOr = mangos + oranges;
+console.log(sumApMa > sumMaOr); //==> true
+
+// is apples subtracted by mangos less than oranges divided by mangos?
+console.log(apples - mangos < oranges / mangos); //==> false
+
+// are oranges equal to mangos and apples?
+console.log(mangos === apples && mangos === oranges && apples === oranges); //==> false
```
Comment (ciamac-da) [inline Elias/exercise-4.js]: In this case you dont need to use && in console.log and the simple answer is 
console.log(apples == oranges == mangoes);

File: Elias/exercise-4.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+//is the remainder of apples divided by mangos equal to the remainder of oranges divided by mangos?
+console.log(apples % mangos === oranges % mangos); //==> true
+
+//is mangos added to apples greater than oranges minus mangos?
+console.log(mangos + apples > oranges - mangos); //==> false
+console.log(mangos + apples === oranges - mangos); //==> true
```
Comment (ciamac-da) [inline Elias/exercise-4.js:37]: You have to get rid of line 37 and add another condition which is 
 console.log((mangoes + apples) >= (oranges - mangoes));
instead!

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Hey Elias. Thanks for pushing. Please read my comments on your code and try to fix the code. Thanks!

File: Elias/exercise-4.js (hunk 1 above)
Comment (GuderianE) [reply Elias/exercise-4.js]: The third question only states if it is greater than, so I fulfilled the task. Right after that however, I made a mistake and there I have to add <=. Thanks for checking out my code.

File: Elias/exercise-4.js (hunk 2 above)
Comment (GuderianE) [reply Elias/exercise-4.js]: Ok got it, thanks.

File: Elias/exercise-4.js (hunk 3 above)
Comment (GuderianE) [reply Elias/exercise-4.js:37]: According to the task, I did what it said just differently but obviously yours is more efficient.
I will add it. Thanks.

**Review by ciamac-da — APPROVED** [review]
💯

**Pull request merged**